build/openwisp_dashboard/load_init_data.py

```diff
@@ -2,16 +2,113 @@
 Creates the admin user when openwisp2 is installed
 """
 
-from django.conf import settings
-from django.contrib.auth import get_user_model
-
+import json
 import os
 import django
 
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp.settings')
 django.setup()
 
 User = get_user_model()
 
-User.objects.filter(is_superuser=True).exists() or \
-    User.objects.create_superuser("admin", "admin@example.com", "admin")
+
+def create_admin():
+    '''
+    Creates superuser `admin` if it does not exist.
+    '''
+    User.objects.filter(is_superuser=True).exists() or \
+        User.objects.create_superuser("admin", "admin@example.com", "admin")
+
+
+def get_vpn_organization(vpn_org):
+    '''
+    Get organization for which VPN server and template
+    needs to be made
+    '''
+    return Organization.objects.get(slug=vpn_org)
+
+
+def create_default_CA(vpnOrg):
+    '''
+    Create default certificate authority
+    '''
+    if not Ca.objects.filter(name='default').exists():
```

Comment:
what happens if a user renames the `default` VPN to something else?
will a new VPN be created?

Reply:
Renaming VPN Server name will not cause any problem, just a new instance of OpenVPN server with the new VPN name will need to be created. ( i.e the environment variable `VPN_NAME` needs to be manually updated in the VPN server instance.)
If you rename the certificate or CA associated (that are name `default`), then everything will keep working except when dashboard instance is restarted for any reason, a CA named default and a Cert named default will be created, both useless but other than that,  everything will keep working.